### PR TITLE
Feature/live preview new

### PR DESCRIPTION
**Pull request by cadw**
Everything works fine, however ONE IMPORTANT PROBLEM!!!
- Bower messed up the files, so I added CSS manually, I have not been able to fix this any other way, if someone has suggestions, please let me know!

### CODE REVIEW

File: src/main/webapp/app/live/live.controller.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        vm.loadCues = loadCues;
+        vm.loadCues();
+
+        function loadCues () {
+            Cue.query({}, onSuccess, onError);
+
+            function onSuccess(data, headers) {
+                vm.cues = data;
+                vm.queryCount = vm.totalItems;
+                console.log(vm.cues[0]);
```
Comment (rubenwiersma) [inline src/main/webapp/app/live/live.controller.js]: Log can go out

File: src/main/webapp/app/live/live.controller.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+     * The controller for the script view.
+     * @param $scope the scope of the map
+     * @param $state the state of the map
+     * @param Camera the camera entity
+     * @param AlertService the alertservice
+     * @constructor
+     */
+    function LiveController ($scope, $state, Cue, AlertService) {
+        var vm = this;
+        vm.loadCues = loadCues;
```
Comment (rubenwiersma) [inline src/main/webapp/app/live/live.controller.js]: I think vm.loadCues = Cue.query(); could work too

File: src/main/webapp/app/live/live.html [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+
+                100% {
+                    -webkit-transform: scale(1);
+                    -moz-transform: scale(1);
+                    -ms-transform: scale(1);
+                    -o-transform: scale(1);
+                    transform: scale(1);
+                }
+            }
+
```
Comment (rubenwiersma) [inline src/main/webapp/app/live/live.html]: This should be in the custom.css or somewhere else before we merge

File: src/test/javascript/karma.conf.js [hunk 4]
```diff
@@ -46,7 +46,8 @@ module.exports = function (config) {
             'src/main/webapp/bower_components/blob-polyfill/Blob.js',
             'src/main/webapp/bower_components/file-saver.js/FileSaver.js',
             'src/main/webapp/bower_components/angular-file-saver/dist/angular-file-saver.bundle.js',
-            'src/main/webapp/bower_components/vis/dist/vis.min.js',
+            'src/main/webapp/bower_components/vis/dist/vis.js',
```
Comment (rubenwiersma) [inline src/test/javascript/karma.conf.js]: We should look into how this is fixed and why it doesn't use vis.min.js

**Comment by hungnguyen94** [comment]
Bower and the CSS should be fixed before we can merge this.

**Comment by cadw** [comment]
I have fixed the issues Ruben mentioned, Hung and I are trying to fix the Bower plugin for this timeline atm.

**Comment by cadw** [comment]
We have decided not to use the plugin, so I have fixed every issue and I am sure it can be merged now.

**Comment by hungnguyen94** [comment]
I didn't get the CSS working either. We could just load it manually. I would name the file `angular-timeline.css` instead of `vertical-timeline.css` though.

**Pull request merged**